ChunkyDocs/docs/plugins/chunky_plugins.md

```diff
@@ -0,0 +1,48 @@
+# Chunky Plugins
+
+---
+
+The functionality of Chunky can be extended with plugins. Plugins can add new blocks, new post-processors, and even new renderer implementations.
+
+[:material-puzzle: Get plugins](../plugin_list){ .md-button }
+
+---
+
+## Installation
+
+Plugins are usually distributed as JAR files. To install a plugin, follow the instructions below.
+
+- Step 1: Download the plugin JAR file.
+
+- Step 2: Move the plugin to the "plugins" directory of the Chunky [settings directory](../../getting_started/configuring_chunky_launcher#advanced-settings). If you do not know where it is located, then skip this step.
```

Comment:
I think we should remove this and only support adding plugins through the plugin manager.

Reply:
Moving a plugin to the "plugins" folder is exactly what the plugin manager does, so there is no reason to remove this step, since it is a simpler method for some users to install plugins.